core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -237,6 +238,12 @@ public static String initcap(String s) {
     return newS.toString();
   }
 
+  /** SQL ASCII(string) function. */
+  public static int ascii(String s) {
+    return s.length() == 0
```

Comment:
Is it just me who prefers  `string.isEmpty()` over `string.size() > 0` ?

Reply:
You are right.